Formula/dsvpn.rb

```diff
@@ -0,0 +1,57 @@
+class Dsvpn < Formula
+  desc "Dead Simple VPN"
+  homepage "https://github.com/jedisct1/dsvpn"
+  url "https://github.com/jedisct1/dsvpn.git",
```

Comment:
Please build from a source tarball, instead of git commit (to cache downloads)

Reply:
Building from release tarballs now.